readthedocs.yml

```diff
@@ -1,16 +1,9 @@
-# .readthedocs.yml
-# Read the Docs configuration file
-version: 2
```

Comment:
is removing this line goes back to V1 ? (and is it a problem?)

Reply:
I'll have to try anyway, as this is not working obviously :D